### PR TITLE
Join table for visits

### PR DESCRIPTION
**Pull request by blase147**
Dean owns Ang and Boa
insert into owners
INSERT INTO SPECIES
Insert species_id value1
Insert species_id value2
Jennfer owns Gabu and Pika
join multiple tables 1
Melody owns Char Sq Bloss
modify inserted animals to include species_id
Sam Smith owns Agumon
add primary key
Animals table
Bob owns Dev and Plan
Create Owners
create species
In this PR
   I created queries for joining tables for visits

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -84,4 +84,128 @@ BEGIN;
 UPDATE animals.animals
 SET owners_id = 'Dean Winchester'
 WHERE name IN ('Angemon', 'Boamon');
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+-- insert records into vets table
+INSERT INTO vets(name, age, date_of_graduation) VALUES('William Tatcher',45,'23/04/2000');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Maisy Smith',26,'17/07/2019');
```
Comment (Deepakdanger) [inline data.sql]: - [ ] Kindly insert the correct graduation date in `Maisy Smith`, It is `'2019-01-17'`

File: data.sql [hunk 2]
```diff
@@ -84,4 +84,128 @@ BEGIN;
...
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Stephanie Mendez',64,'04/05/1981');
+INSERT INTO vets(name, age, date_of_graduation) VALUES('Jack Harkness',38,'08/07/2008');
```
Comment (Deepakdanger) [inline data.sql]: - [ ] Kindly insert the correct graduation date in `Jack Harkness`, It is `'2008-06-08'`

File: data.sql [hunk 3]
```diff
@@ -84,4 +84,128 @@ BEGIN;
...
+
+-- insert records into specialization table based on conditions
+INSERT INTO specializations(vets_id, species_id) VALUES(
+    (SELECT id FROM vets WHERE name = 'William Tatcher'),
+    (SELECT id FROM species WHERE name = 'Pokemon')
+);
+
```
Comment (Deepakdanger) [inline data.sql:100]: - [ ] There should be one more insertion.
```sql
INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));
```

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @blase147 , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @blase147, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Website looks professional

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**